src/components/JuzMarker/decoration.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { juzAndHizbArabicNum } from './markNumbers';
+
+const juzMarkTop = require('../../../static/images/juzTopArt.svg');
+const juzMarkBottom = require('../../../static/images/juzBottomArt.svg');
+
+const StyledContainer = styled.div`
+  padding: 0.05em;
```

Comment:
`em` is hard to control cause it's context dependent. Can we change this to be a fixed px?

Reply:
i didn't know that
Thanks will work on it